### DISCUSSION TITLE
LSF - are the retries working?

### DISCUSSION THREAD

**Discussion by complexgenome** [nextflow-io/nextflow#4182] [Q&A]
Hi
I have set retries in my processes at LSF scheduler. I am not sure why retries aren't working. Please see attached message for retries. 
<img width="1234" alt="Screen Shot 2023-08-12 at 8 29 32 PM" src="https://github.com/nextflow-io/nextflow/assets/5683969/fb114130-543e-4f7c-a190-773e454f342b">

I've set applybqsr process as:

```
process applybqsr{

        maxForks 3
        debug true
        errorStrategy 'retry'
    maxRetries 10

```

In the config file I've put the following:

```
process {
        withName: applybqsr {
                memory={'60 GB'* task.attempt }
                 cpus = { 2 * task.attempt }
        }
}


```

How do I make retries to work successfully? It serves no purpose if they fail after retries defined in process as I've to keep checking, fix memory requirement.

**Comment by bentsherman** [comment]
It does retry, but fails when submitting the job to the scheduler. I would check the retried task, maybe try to submit it manually and see if you get an error from the scheduler.
